alacritty_terminal/src/grid/mod.rs

```diff
@@ -601,13 +601,34 @@ impl<'a, T> Iterator for GridIterator<'a, T> {
         match self.point {
             Point { column, .. } if column == self.grid.last_column() => {
                 self.point.column = Column(0);
-                self.point.line += 1;
+                self.point.line += Line(1);
```

Comment:
This isn't really an improvement, there's a reason why Add/SubAssign is implemented for both.

Reply:
Other than for this spot all the additions seem to be done explicitly using the wrapper types (for example two lines below this one), so I figured it was a missed styling